.circleci/config.yml

```diff
@@ -44,6 +53,12 @@ jobs:
       - store_artifacts:
           path: ./docs/build/html
 
+      - save_cache:
+          key: tutorial-{{ checksum "/tmp/tutorial_files.md5" }}
+          paths:
+            - tutorial/MNIST
```

Comment:
```suggestion
```

Thank you for the explanation. So, we can remove it for now, right?

Reply:
Absolutely.